### ISSUE TITLE
Client Settings tab Key Bindings is a black hole when using a keyboard

### ISSUE THREAD

**Issue by sixlettervariables** [MegaMek/megamek#1294]
### Environment
What version of MegaMek does your issue apply to?  0.45.X
What operating system are you using?  Windows
What java version are you using?  8

### Description
If you're navigating with a keyboard, you get stuck once you tab through to the Key Bindings tab under Client Settings. The first "modifiers" text box takes focus and captures keys like TAB.

**Comment by sixlettervariables** [comment]
I've attempted a fix this, but wasn't successful. I had a thought to improve this: require the enter key to be pressed to "set" the key. This would allow you to TAB around (and out) of that tab.
